src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -374,8 +373,8 @@
                   <columns>
                     <TableColumn fx:id="tableColumnCandidateName" minWidth="20.0"
                       prefWidth="230.0" text="Name"/>
-                    <TableColumn fx:id="tableColumnCandidateCode" minWidth="20.0"
-                      prefWidth="100.0" text="Code"/>
+                    <TableColumn fx:id="tableColumnCandidateAliases" minWidth="20.0"
+                      prefWidth="200.0" text="Aliases / Code" />
```

Comment:
Nit: "Codes" since "Aliases" is plural.

Reply:
I don't have super strong opinions here, but do want to flag this was also intentional: although there is no difference to us about codes and aliases, in the CVRs that use a `code`, there can be only one `code`.

I did make the change in this case, but let me know if you prefer it to remain singular after hearing this explanation.